### PR TITLE
Task-39254: Dlp Quarantine Page UpgradePlugin

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: data-upgrade-pages/src/main/resources/conf/portal/configuration.xml [hunk 1]
```diff
@@ -156,5 +156,52 @@
       </init-params>
     </component-plugin>
   </external-component-plugins>
+
+  <external-component-plugins>
+    <target-component>org.exoplatform.commons.upgrade.UpgradeProductService</target-component>
+    <component-plugin>
+      <name>DlpQuarantinePagesUpgradePlugin</name>
+      <set-method>addUpgradePlugin</set-method>
+      <type>org.exoplatform.agenda.migration.PagesMigration</type>
```
Comment (azayati) [inline data-upgrade-pages/src/main/resources/conf/portal/configuration.xml]: I think this need to be renamed (Removing agenda from the package)

**Review by azayati — CHANGES_REQUESTED** [review]
See review cc @rdenarie

File: data-upgrade-pages/src/main/resources/conf/portal/configuration.xml (hunk 1 above)
Comment (rdenarie) [reply data-upgrade-pages/src/main/resources/conf/portal/configuration.xml]: +1

**Review by azayati — APPROVED** [review]
Please add the task ID for the PR title and commit

File: data-upgrade-pages/src/main/java/org/exoplatform/migration/PagesMigration.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package org.exoplatform.agenda.migration;
+package org.exoplatform.migration;
```
Comment (rdenarie) [inline data-upgrade-pages/src/main/java/org/exoplatform/migration/PagesMigration.java:1]: Why do you change package Name ?

File: data-upgrade-pages/src/main/resources/conf/portal/configuration.xml [hunk 3]
```diff
@@ -26,7 +26,7 @@
     <component-plugin profiles="agenda">
       <name>CalendarPagesUpgradePlugin</name>
       <set-method>addUpgradePlugin</set-method>
-      <type>org.exoplatform.agenda.migration.PagesMigration</type>
+      <type>org.exoplatform.migration.PagesMigration</type>
```
Comment (rdenarie) [inline data-upgrade-pages/src/main/resources/conf/portal/configuration.xml:29]: not needed

File: data-upgrade-pages/src/main/resources/conf/portal/configuration.xml [hunk 4]
```diff
@@ -69,7 +69,7 @@
     <component-plugin profiles="agenda">
       <name>CalendarWidgetPagesUpgradePlugin</name>
       <set-method>addUpgradePlugin</set-method>
-      <type>org.exoplatform.agenda.migration.PagesMigration</type>
+      <type>org.exoplatform.migration.PagesMigration</type>
```
Comment (rdenarie) [inline data-upgrade-pages/src/main/resources/conf/portal/configuration.xml:72]: not needed

File: data-upgrade-pages/src/main/resources/conf/portal/configuration.xml [hunk 5]
```diff
@@ -115,7 +115,7 @@
     <component-plugin>
       <name>TasksPagesUpgradePlugin</name>
       <set-method>addUpgradePlugin</set-method>
-      <type>org.exoplatform.agenda.migration.PagesMigration</type>
+      <type>org.exoplatform.migration.PagesMigration</type>
```
Comment (rdenarie) [inline data-upgrade-pages/src/main/resources/conf/portal/configuration.xml:118]: not needed

File: data-upgrade-pages/src/test/java/org/exoplatform/migration/PagesMigrationTest.java [hunk 6]
```diff
@@ -1,4 +1,4 @@
-package org.exoplatform.agenda.migration;
+package org.exoplatform.migration;
```
Comment (rdenarie) [inline data-upgrade-pages/src/test/java/org/exoplatform/migration/PagesMigrationTest.java:1]: not needed

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Review by rdenarie — APPROVED** [review]

**Pull request merged**